heatmap/src/window.rs

```diff
@@ -11,6 +13,22 @@ pub struct Window<'a> {
     pub(crate) histogram: &'a Histogram,
```

Comment:
As a suggestion, maybe make this a `Cow<'a, &'a Histogram>` and then implement [`ToOwned`](https://doc.rust-lang.org/stable/std/borrow/trait.ToOwned.html) for `&Histogram` and `Window`? Then you wouldn't need a separate `OwnedWindow` type.

Reply:
Good idea! Will leave for now and come back to that.